SPECS/ocaml-base/ocaml-base.spec

```diff
@@ -3,19 +3,34 @@
 # tests at all until we are able to add ppx_jane.
 %global srcname base
 
+# This package is needed to build ppx_jane, but its tests require ppx_jane.
+# Break the dependency cycle here.
+%bcond_with test
```

Comment:
A test-only dependency shouldn't cause cyclic dependencies as long as the BR is behind the `%{with_check}` macro.

Reply:
Changed it